src/main/java/ch/njol/skript/expressions/ExprColorOf.java

```diff
@@ -224,14 +190,53 @@ public String toString(@Nullable Event event, boolean debug) {
 		return "color of " + getExpr().toString(event, debug);
 	}
 
-	@Nullable
-	private Colorable getColorable(Object colorable) {
+	private Consumer<TextDisplay> getDisplayChanger(ChangeMode mode, @Nullable Color[] colors) {
```

Comment:
```suggestion
	private Consumer<TextDisplay> getDisplayChanger(ChangeMode mode, Color @Nullable [] colors) {
```
applies to all the changers